mibs/USHA-MIB

```diff
@@ -0,0 +1,2620 @@
+-- Ingrasys USHA MIB
```

Comment:
Could you please put the mib in a directory ?

Reply:
> Could you please put the mib in a directory ?

Hi @PipoCanaja 
mi initial idea was to put usha-mib next to ups-mib, as they are both general mib files, but OK
as you wish, moved MIB to separate folder

thank you for review

Reply:
USHA-MIB isn't general/generic? It contains a specific enterprise OID from Ingrasys:

 { iso org(3) dod(6) internet(1) private(4) enterprises(1) ingrasys(2468)

Reply:
> USHA-MIB isn't general/generic? 

hi @rudybroersma 

i saw that few manufacturer using usha-mib for their cards, that was a reason to put it in mibs root
but as @PipoCanaja requested, it is moved